code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -69,8 +69,8 @@
 
 /obj/item/reagent_containers/hypospray/safety
 	name = "medical hypospray"
-	desc = "A modified hypospray for quickly injecting safe medicinal chemicals."
-	icon_state = "combat_hypo"
+	desc = "An uninspiring general use medical hypospray for quick injection of chemicals. This model seems a bit outdated."

```

Comment:
What. It's not outdated; the fact it actually has additional safeties and behaviors doesn't mean it's out-dated at all.

Also, "uninspiring"? How does this benefit the description at all? It's neither comical nor informative to the user, nor a reference to anything; it's just kinda a jab at the device because...?

Reply:
The intent was to make it more obvious that the medical vending machine hypospray was _not_ an item worth stealing as a syndicate should there ever be confusion hence the "It appears outdated" line, and the word uninspiring, which I meant to make it pass off as a not that valuable item. Regarding if the hypospray is _actually_ newer than the CMO's and or Syndicate Injectors is up to the lore team.